subprojects/docs/src/docs/release/notes.md

```diff
@@ -46,6 +46,24 @@ Garbage collection time goes from [2.6 seconds](https://scans.gradle.com/s/3bg67
 
 While the impact on your build may vary, most builds can expect a noticeably shorter feedback loop when editing Kotlin DSL build logic thanks to this improvement.
 
+### Improved cache hits with normalized runtime classpaths
+
+For [up-to-date checks](userguide/more_about_tasks.html#sec:up_to_date_checks) and the [build cache](userguide/build_cache.html), Gradle needs to determine if two task input properties have the same value. In order to do so, Gradle first normalizes both inputs and then compares the result.
```

Comment:
I'd also link to https://docs.gradle.org/current/userguide/more_about_tasks.html#sec:configure_input_normalization when mentioning normalization for the first time.

Reply:
Done.